contracts/CryptoHamster/test/constants/config.json

```diff
@@ -0,0 +1,17 @@
+{
+	"host": "http://localhost:3001/",
+	"internalHost": "http://localhost:3001/internal/",
+	"ownerKeyPair": {
+		"secretKey": "bb9f0b01c8c9553cfbaf7ef81a50f977b1326801ebf7294d1c2cbccdedf27476e9bbf604e611b5460a3b3999e9771b6f60417d73ce7c5519e12f7e127a1225ca",
+		"publicKey": "ak_2mwRmUeYmfuW93ti9HMSUJzCk1EYcQEfikVSzgo6k2VghsWhgU"
+	},
+	"notOwnerKeyPair": {
+		"secretKey": "7fa7934d142c8c1c944e1585ec700f671cbc71fb035dc9e54ee4fb880edfe8d974f58feba752ae0426ecbee3a31414d8e6b3335d64ec416f3e574e106c7e5412",
+		"publicKey": "ak_tWZrf8ehmY7CyB1JAoBmWJEeThwWnDpU4NadUdzxVSbzDgKjP"
+	},
+	"notOwnerPubKeyHex": "0xdf473b854e8d78394c20abfcb8fda9d0ed5dff8703d8668dccda9be157a60b6d",
```

Comment:
please check if this hex is the right one for the address. I think it should be this: 0x74f58feba752ae0426ecbee3a31414d8e6b3335d64ec416f3e574e106c7e5412

but please check. And also if the hex is wrong change it on all currencies